src/metabase/api/advanced_computation/public.clj

```diff
@@ -37,4 +133,55 @@
                                out-chan)))
              nil))))
 
+(api/defendpoint ^:streaming GET "/pivot/dashboard/:uuid/card/:card-id"
+  "Fetch the results for a Card in a publicly-accessible Dashboard. Does not require auth credentials. Public
+   sharing must be enabled."
+  [uuid card-id parameters]
+  {parameters (s/maybe su/JSONString)}
+  (api/check-public-sharing-enabled)
+
+  (let [dashboard-id (api/check-404 (db/select-one-id Dashboard :public_uuid uuid, :archived false))]
+    (public-dashcard-results-async dashboard-id card-id :api parameters)))
+
+;; /embed/card/:token/query
+(api/defendpoint ^:streaming GET "/pivot/embed/card/:token/query"
+  "Fetch the results of running a Card using a JSON Web Token signed with the `embedding-secret-key`.
+
+   Token should have the following format:
+
+     {:resource {:question <card-id>}
+      :params   <parameters>}"
+  [token & query-params]
+  (run-query-for-unsigned-token-async (eu/unsign token) :api query-params))
+
+;; /preview_embed/card/:token/query
+(api/defendpoint ^:streaming GET "/pivot/preview_embed/card/:token/query"
```

Comment:
I think it's a little confusing to have embed endpoints and public endpoints mixed together in the same namespace